Formula/gerbil-scheme.rb

```diff
@@ -25,16 +26,23 @@ class GerbilScheme < Formula
   depends_on "leveldb"
   depends_on "libyaml"
   depends_on "lmdb"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   uses_from_macos "libxml2"
   uses_from_macos "sqlite"
 
+  on_macos do
+    depends_on "gcc"
+  end
+
+  fails_with :clang do
+    cause "gambit-scheme is built with GCC"
+  end
```

Comment:
`gambit-scheme` appears to only use C, so continuing to compile this with `clang` will probably work (unless there are also performance problems here).

Reply:
One thing was the previous hang was on `gerbil-scheme`. Also, GCC optimization flags makes Clang harder to use (but it seems recommended by upstream and used by other repositories).